types/crypto.go

```diff
@@ -25,9 +25,9 @@ func (s Signature) VerifyByOperators(data MessageSignature, domain DomainType, s
 	for _, id := range data.GetSigners() {
 		found := false
 		for _, n := range operators {
-			if id == n.GetID() {
+			if id == n.OperatorID {
```

Comment:
All the BLS signature funcs here are not used

Reply:
That's good, no?

Reply:
functions that are not used, remove them and tests

Reply:
This function was deleted since it isn't in use anymore